data_model/src/transaction.rs

```diff
@@ -177,6 +176,12 @@ model! {
     );
 }
 
+impl core::fmt::Debug for WasmSmartContract {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        write!(f, "WASM binary(len = {})", self.0.len())
+    }
+}
```

Comment:
why?

Reply:
Yes, it's an interesting point, thank you for mentioning.

So `CustomDebug` from `derive_more` didn't produced what we wanted. I noticed this while debugging and checked it with expansion. And that manual implementation fixes the problem.
I don't know why it's happened and why it's not happening for other structs. May be you can check it on your own machine, if you're not sure?

Reply:
Now I changed it to follow #3340